cmd/mattermost/commands/user.go

```diff
@@ -368,6 +380,48 @@ func userCreateCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func usersToBots(args []string, a *app.App) error {
```

Comment:
Here you return an `error` but in the code there is only one return, and is a `nil`, so I guess this function never "raises" an error.